src/main/generated/io/vertx/ext/stomp/FrameConverter.java

```diff
@@ -0,0 +1,43 @@
+package io.vertx.ext.stomp;
```

Comment:
Take a look at the headers for generated classes that we do for Rx (for example).